src/viur/core/render/json/default.py

```diff
@@ -85,19 +85,29 @@ def renderSingleBoneValue(self, value: Any,
         :return: A dict containing the rendered attributes.
         """
         if isinstance(bone, bones.RelationalBone):
-            if isinstance(value, dict):
-                return {
-                    "dest": self.renderSkelValues(value["dest"], injectDownloadURL=isinstance(bone, bones.FileBone)),
-                    "rel": (self.renderSkelValues(value["rel"], injectDownloadURL=isinstance(bone, bones.FileBone))
-                            if value["rel"] else None),
-                }
+            if not isinstance(value, dict):
+                return None  # FIXME: Shall I raise?
+
+            dest = value["dest"]
+            rel = value["rel"]
+
+            if bone.expand:
+                skel = skeleton.skeletonByKind(bone.kind)()
+                if not skel.fromDB(dest["key"]):
+                    return None  # FIXME: Shall I raise?
```

Comment:
Yeah, of course! Otherwise there wouldn't be a difference between no and invalid data!